prow/pjutil/filter.go

```diff
@@ -29,6 +29,9 @@ import (
 
 var TestAllRe = regexp.MustCompile(`(?m)^/test all,?($|\s.*)`)
 
+// TestRe provides the regex for `/test job-name`
+var TestRe = regexp.MustCompile(`/test\s+(\w+)`)
```

Comment:
This regular expression won't match all the valid `/test` commands.  I don't think we technically have any constraints on what can be used for the `trigger` and `rerun_command` fields so they don't even necessarily start with `/test`: https://docs.prow.k8s.io/docs/jobs/#how-to-configure-new-jobs

We'll obviously need to start enforcing some constraint on our Prow instance (probably just require a `/test` prefix) if we want to skip unrelated changes before loading the config, but we'll need to make this regexp configurable and optional rather than hard coding it to avoid a breaking change for Prow instances that we don't manage. We'll also need to notify our users and begin enforcing this constraint on Gerrit ProwJobs (not GitHub jobs) via the `checkconfig` tool which runs presubmit config validation.

Reply:
Still need to get rid of `TestRe`.

Reply:
Done.

Reply:
This is not done. Please add the requested validation to the `checkconfig` tool.